addons/crm/models/crm_lead.py

```diff
@@ -913,14 +913,15 @@ def get_empty_list_help(self, help):
             if alias_record and alias_record.alias_domain and alias_record.alias_name:
                 email = '%s@%s' % (alias_record.alias_name, alias_record.alias_domain)
                 email_link = "<a href='mailto:%s'>%s</a>" % (email, email)
-                dynamic_help = _("""All email incoming to %s will automatically
-                    create new opportunity. Update your business card, phone book, social media,...
-                    Send an email right now and see it here.""") % (email_link,)
-                return '<p class="oe_view_nocontent_create">%s</p>%s<p>%s</p>' % (_('Click to add a new opportunity'), help, dynamic_help)
+                if self._context.get('default_type') == 'lead':
+                    help_title = _('Create a new lead or send an email to %s') % (email_link)
+                else:
+                    help_title = _('Create a new opportunity to add it to your pipeline')
+                return '<p class="oe_view_nocontent_create">%s</p><p>%s</p>' % (help_title, help)
```

Comment:
Side note:
```python
help
```
is a [Python builtin function](https://docs.python.org/3/library/functions.html#help) (not the most used in a non-interactive context, I agree), but as a general guideline: we should not override Python builtins